src/meshFeature.js

```diff
@@ -349,20 +345,37 @@ var meshFeature = function (arg) {
      * used.  This could leave vertices that are unused by any element, but
      * removing those is expensive so it is not done. */
     if (vertexValueFuncs.used) {
-      var remap = new Array(numPts),
-          vpe = result.verticesPerElement;
-      for (i = usedPts = 0; i < numPts; i += 1) {
-        idx = result.index[i];
-        if (vertexValueFuncs.used(data[idx], idx)) {
-          remap[i] = usedPts;
-          result.index[usedPts] = result.index[i];
-          usedPts += 1;
-        } else {
-          remap[i] = -1;
+      for (i = 0; i < numPts; i += 1) {
+        idx = result.index ? result.index[i] : i;
+        if (!vertexValueFuncs.used(data[idx], idx)) {
+          break;
         }
       }
-      result.index.splice(usedPts);
-      if (usedPts !== numPts) {
+      if (i !== numPts) {
```

Comment:
whats changed here?

Reply:
Some of the mesh feature was refactored to use less memory by not duplicate arrays if not needed.  `usedPts` is no longer computed while generating a preliminary array, as by skipping that array things are faster.  If some vertices are unused, then the previous for loop will have stopped before reaching `numPts`.  In this case we go into the next conditional and exclude the unused vertices.